condensedinlinepanel/edit_handlers.py

```diff
@@ -108,6 +109,7 @@ def get_form_extra_data(form):
             'forms': [
                 {
                     'id': i,
+                    'as_str': str(form.instance),
```

Comment:
Variables on the form should be camel case. Also, I think a name should make it clear this is for the existing database instance. How about ``instanceAsStr``?

We should also add this to the ``Form`` interface in ``types.ts``